crates/rolldown_utils/src/sanitize_file_name.rs

```diff
@@ -0,0 +1,21 @@
+use std::borrow::Cow;
+
+// TODO support `output.sanitizeFileName`
+// Follow from https://github.com/rollup/rollup/blob/master/src/utils/sanitizeFileName.ts
```

Comment:
I quickly look the implementation, it seems this is not a fully equivalent port?